kyverno-policy-reporter-ui.advisories.yaml

```diff
@@ -3,7 +3,6 @@ schema-version: 2.0.2
 package:
   name: kyverno-policy-reporter-ui
 
-advisories:
```

Comment:
Don't we still need an `advisories:` key/field?

Reply:
bad rebase, thanks for the catch, done PTAL